tlatools/org.lamport.tlatools/test/tla2sany/TestLevelChecking.java

```diff
@@ -0,0 +1,155 @@
+package tla2sany;
```

Comment:
What is the test coverage of the new code?

Reply:
What do you mean?

Reply:
Oh, you mean like is the new table and the logic reading it in exercised. Yes that is tested.